af/analysis/analysis/analysis.py

```diff
@@ -942,11 +945,140 @@ def generate_slow_query_summary(conf):
+    to detect and alert on inconsistencies
+
+    Runs in a dedicated thread and writes in its own .prom file
+
+    This is the most important function, therefore it pings the SystemD watchdog
+    """
+    log.info("Started monitor_measurement_creation thread")
+    # TODO: switch to OOID
+
+    INTERVAL = 60 * 5  # half of the watchdog interval in the analysis.service file
+    nodeexp_path = "/run/nodeexp/db_metrics.prom"
+    if no_systemd == False:
+        watchdog = sdnotify.SystemdNotifier()
+
+    prom_reg = prom.CollectorRegistry()
+    gauge_family = prom.Gauge(
+        "measurements_flow",
+        "Measurements being created",
+        labelnames=["type"],
+        registry=prom_reg,
+    )
+    queries = dict(
+        fastpath_count="""SELECT COUNT(*)
+            FROM fastpath
+            WHERE measurement_start_time > %(since)s
+            AND measurement_start_time <= %(until)s
+        """,
+        pipeline_count="""SELECT COUNT(*)
+            FROM measurement
+            WHERE measurement_start_time > %(since)s
+            AND measurement_start_time <= %(until)s
+        """,
+        pipeline_not_fastpath_count="""SELECT COUNT(*)
+        FROM measurement
+        LEFT OUTER JOIN input ON input.input_no = measurement.input_no
+        JOIN report ON report.report_no = measurement.report_no
+        WHERE NOT EXISTS (
+            SELECT
+            FROM fastpath fp
+            WHERE measurement_start_time > %(since_ext)s
+            AND measurement_start_time <= %(until_ext)s
+            AND fp.report_id = report.report_id
+            AND fp.test_name = report.test_name
+            AND fp.input = input.input
+        )
+        AND measurement_start_time > %(since)s
+        AND measurement_start_time <= %(until)s
+        """,
+        fastpath_not_pipeline_count="""SELECT COUNT(*)
+        FROM fastpath fp
+        WHERE NOT EXISTS (
+            SELECT
+            FROM measurement
+            LEFT OUTER JOIN input ON input.input_no = measurement.input_no
+            JOIN report ON report.report_no = measurement.report_no
+            WHERE measurement_start_time > %(since_ext)s
+            AND measurement_start_time <= %(until_ext)s
+            AND fp.report_id = report.report_id
+            AND fp.test_name = report.test_name
+            AND fp.input = input.input
+        )
+        AND measurement_start_time > %(since)s
+        AND measurement_start_time <= %(until)s
+        """,
+    )
+
+    # test connection and notify systemd
+    conn, _ = setup_database_connections(conf.standby)
+    with conn.cursor() as cur:
+        cur.execute("SELECT 1")
+    conn.close()
+    if no_systemd == False:
+        watchdog.notify("READY=1")
+
+    cycle_seconds = 0
+
+    while True:
+        try:
+            log.info("Gathering fastpath count")
+            conn, dbengine = setup_database_connections(conf.standby)
+            delta = timedelta(minutes=5)
```

Comment:
Maybe `5` should be a variable which is in sync with `INTERVAL = 60 * 5`?

Reply:
No, this is independent from the run interval.